### PR TITLE
added diamond item and grab speed based on size

### PR DESCRIPTION
**Pull request by geygrill**

### CODE REVIEW

File: lib/Display/display.cpp [hunk 1]
```diff
@@ -62,13 +62,15 @@ void display::resetSkyLeft(int xLocation)
...
-    for(int i = 0; i < 6; i++) {
+    for(int i = 0; i < 9; i++) {
         Item item = items[i];
         if (item.type == GOLD){
             tft.fillRect(item.x, item.y, item.size, item.size, COLOR_GOLD);
         }
         else if(item.type == STONE) {
             tft.fillRect(item.x, item.y, item.size, item.size, COLOR_ROCK);
+        } else if(item.type == DIAMOND) {
+            tft.fillRect(item.x, item.y, item.size, item.size, COLOR_DIAMOND);
```
Comment (enigma999) [inline lib/Display/display.cpp:73]: use switch case https://www.w3schools.com/cpp/cpp_switch.asp

File: lib/Hook/hook.cpp [hunk 2]
```diff
@@ -36,10 +36,21 @@ void hook::calculateAndDrawHook(int xOrigin, int yOrigin, float angle, Item item
...
+                    } else if (item.size < 20) {
+                        _delay_ms(10);
+                    } else if (item.size < 30) {
+                        _delay_ms(20);
+                    } else if (item.size < 40) {
+                        _delay_ms(40);
+                    } else {
+                        _delay_ms(80);
+                    }
+
```
Comment (enigma999) [inline lib/Hook/hook.cpp:51]: make this a function like item.size/3. so you have a solution that is more gradual and works for all sizes.

File: src/main.cpp [hunk 3]
```diff
@@ -37,17 +37,19 @@ int testArray[ARRAY_SIZE] = {1, 1, 1, 0, 0, 0, 1, 1, 1, 0, 0, 0, 1, 1, 1, 0}; //
...
+Item stone1(STONE, 10, 200, 15);
+Item stone2(STONE, 50, 110, 15);
+Item stone3(STONE, 240, 110, 50);
 
-Item items[] = {gold1, gold2, gold3, gold4, stone1, stone2, stone3, stone4};
+Item diamond1(DIAMOND, 260, 210, 5);
+Item diamond2(DIAMOND, 200, 200, 5);
+Item diamond3(DIAMOND, 40, 220, 5);
+
+Item items[] = {gold1, gold2, gold3, stone1, stone2, stone3, diamond1, diamond2, diamond3};
```
Comment (enigma999) [inline src/main.cpp:52]: this should eventualy be a randomizer not hardcoded.

File: lib/Hook/hook.cpp (hunk 2 above)
Comment (geygrill) [reply lib/Hook/hook.cpp:51]: hou je bek

Comment (enigma999) [reply lib/Hook/hook.cpp:51]: :(

**Review by enigma999 — APPROVED** [review]

**Pull request merged**